devito/ir/clusters/algorithms.py

```diff
@@ -84,7 +84,7 @@ class Schedule(QueueStateful):
 
     @timed_pass(name='schedule')
     def process(self, clusters):
-        return self._process_fdta(clusters, 1)
+        return self._process_fatd(clusters, 1)
```

Comment:
Was really just a typo? 😮

Reply:
Not kidding was just wondering if was intentional to have `fdta` or if was meant to be `fatd` from the beginning but wasn't noticed. anyway all good